swagger/swagger-generator/generator-jaxrs/src/main/java/io/servicecomb/swagger/generator/jaxrs/JaxrsSwaggerGeneratorContext.java

```diff
@@ -111,6 +113,8 @@ protected void initParameterAnnotationMgr() {
 
     parameterAnnotationMgr.register(HeaderParam.class, new HeaderParamAnnotationProcessor());
     parameterAnnotationMgr.register(QueryParam.class, new QueryParamAnnotationProcessor());
+
+    parameterAnnotationMgr.register(RawJsonRequestBody.class, new RawJsonRequestBodyAnnotationProcessor());
```

Comment:
maybe, it can be done in AbstractSwaggerGeneratorContext

Reply:
Done